src/client.rs

```diff
@@ -84,73 +86,102 @@ pub struct Client {
     server_url: String,
     bucket_name: String,
     collection_name: String,
-    verifier: Box<dyn Verification> 
+    verifier: Box<dyn Verification>,
 }
 
 impl Default for Client {
     fn default() -> Self {
-        Client {
+        #[cfg(feature = "openssl_verifier")]
```

Comment:
Why not rely on `Client::instantiate_verifier(None)` here! It would limit the usage of `cfg(feature)`

Reply:
done